src/diric.jl

```diff
@@ -0,0 +1,47 @@
+export diric
```

Comment:
Just a comment on code organisation for @martinholters to decide on.

Given that this function is documented in `utils` should the code go in `src/utils.jl`? Similarly, for the tests?

Reply:
Ops I meant @galenlynch sorry

Reply:
It's up to you of course, but personally I prefer to keep unrelated methods in different files.

Reply:
As I've commented a lot on this PR but not on this particular question it might be obvious, but just so it be said: I don't care.